docs/source/reference/1-commcare-cloud/1-installation.rst

```diff
@@ -117,10 +117,7 @@ If you get to this point, congratulations! ``commcare-cloud`` is installed.
 Manual Installation
 -------------------
 
-You will need python 3.10 installed to follow these instructions. See
-:ref:`installation/2-manual-install:Upgrade to Python 3.10` for instructions on
```

Comment:
Did the target of this link get removed earlier? I could not find it.

Reply:
yeah I think it was removed [here](https://github.com/dimagi/commcare-cloud/pull/6070/files)